### PR TITLE
refactor(plugin-lava-player):  add lyrics plugin

### PR DESCRIPTION
**Pull request by guilhermehccruz**
Implements the integration with lavalink lyrics endpoint along with examples of how to use it

### CODE REVIEW

**Comment by samarmeena** [comment]
@guilhermehccruz hi, I wanted to see if you'd be available to discuss a few adjustments on our Discord server. It would be easier to communicate there. If that doesn't work for you, I can update your PR myself. Thanks!

**Comment by guilhermehccruz** [comment]
> @guilhermehccruz hi, I wanted to see if you'd be available to discuss a few adjustments on our Discord server. It would be easier to communicate there. If that doesn't work for you, I can update your PR myself. Thanks!

No problem, I'm `jack3580` in discord

**Comment by samarmeena** [comment]
Your PR is now ready. However, there are a few concerns we need to address before merging:

1. I set up the plugins and enabled YouTube lyrics, but it consistently says "no response" even when lyrics are typically available on the video. I might not have configured it correctly, so it needs to be tested before merging.

2. By default, I designed the plugin-lava-player to work with the YouTube plugin, as it's the most common use case. However, we can't add these features without providing a configuration option to disable them (lyrics slash command at least). This needs to be addressed before moving to production.

3. Lavalink supports numerous other plugins, and we can't integrate them all without first addressing point 2. Users who utilize plugins beyond YouTube should implement their own solutions on top of the queue for these features, as the plugin-lava-player currently only supports the YouTube plugin.

Please review these concerns when you have time. Once resolved, we can proceed with the merge. Thank you.

File: packages/plugin-lava-player/src/queue.ts [hunk 1]
```diff
@@ -39,6 +41,43 @@ export class MusicQueue extends Queue {
     );
   }
 
+  get http() {
```
[the remark this answers is not in the document]
Comment (samarmeena) [reply packages/plugin-lava-player/src/queue.ts:44]: weird error, when I try to do that. You can see that by trying to add this http getter. Let me know if you can resolve it

[the remark this answers is not in the document]
Comment (guilhermehccruz) [reply packages/plugin-lava-player/src/queue.ts]: Got the error, but then it worked on second try, I don't know why

**Pull request merged**